### PR TITLE
Update component “dialogs”

### PR DESCRIPTION
**Pull request by itsmemin**
Automatically generated by Netlify CMS

### CODE REVIEW

File: src/_areas/design-system/components/dialogs.md [hunk 1]
```diff
@@ -1,122 +1,118 @@
 ---
+title: Dialogs
+subtitle: Display an important message or decision in it's own window in front
+  of the current page
```
Comment (dannoye) [inline src/_areas/design-system/components/dialogs.md:4]: Display an important message or decision in its own window in front
  of the current page

File: src/_areas/design-system/components/dialogs.md [hunk 2]
```diff
@@ -1,122 +1,118 @@
...
-   The primary action should always appear on the far right.
+- - -
 
-2. **Secondary actions**  
-   Secondary actions appear to the left of the primary action, 30px below the content area
+## Types and states
 
-3. **Tertiary action**  
-   Tertiary actions or help text appear left aligned 30px below the content
+There are 3 sizes of Dialogs, small, standard and large.
```
Comment (dannoye) [inline src/_areas/design-system/components/dialogs.md:50]: There are 3 sizes of Dialogs: small, standard and large.

**Comment by p-ob** [comment]
I know the conversation happened already, and I think we all agree with the changes to dialog to enable better scrolling / sticky footers. Do we want to add some sort of disclaimer to this documentation, however, that the toolkit has not been update to reflect some of these?

@dave52 has the first step in a testing phase to enable us to address this, so I'm not worried that we'll unlock this feature. Just know we'll likely get some questions that we could get ahead of.

**Aside**: We ought to document "what should I do if design docs and toolkit differ" and strongly emphasize teams still use the toolkit, and let us resolve discrepancies.

File: src/_areas/design-system/components/dialogs.md [hunk 3]
```diff
@@ -1,122 +1,118 @@
...
-<br>
+![](/images/dialog_anatomy.svg)
 
-### Title
+</br> 
 
-H1 title should not span more than 1 line.
+1. **Title:** Ideally, the H1 title in a Dialog should not span more than one line. The title gives the user an understanding of the message or decision they are being presented with. 
+2. **Content area:** More detailed information is contained within the content section of a Dialog. This section is a minimum of 120px tall and has varying widths for the different types of Dialogs.  
+3. **Footer:** The footer is where all the actions are located within a Dialog. Optionally, the footer may remain sticky when the length of the Dialog exceeds the height of the viewport so that the available actions remain visible to the user at all times. More information on scrolling and sticky footers can be found in behavior and best practices.
```
Comment (p-ob) [inline src/_areas/design-system/components/dialogs.md:28]: I know the conversation happened already, and I think we all agree with the changes to dialog to enable better scrolling / sticky footers. Do we want to add some sort of disclaimer to this documentation, however, that the toolkit has not been update to reflect some of these?

@dave52 has the first step in a testing phase to enable us to address this, so I'm not worried that we'll unlock this feature. Just know we'll likely get some questions that we could get ahead of.

**Aside**: We ought to document "what should I do if design docs and toolkit differ" and strongly emphasize teams still use the toolkit, and let us resolve discrepancies.

File: src/_areas/design-system/components/dialogs.md [hunk 4]
```diff
@@ -1,122 +1,118 @@
...
 
-<br>
+![](/images/dialog_anatomy.svg)
 
-### Title
+</br> 
 
-H1 title should not span more than 1 line.
+1. **Title:** Ideally, the H1 title in a Dialog should not span more than one line. The title gives the user an understanding of the message or decision they are being presented with. 
+2. **Content area:** More detailed information is contained within the content section of a Dialog. This section is a minimum of 120px tall and has varying widths for the different types of Dialogs.
```
Comment (juliexiong) [inline src/_areas/design-system/components/dialogs.md:27]:
```suggestion
2. **Content area:** More detailed information is contained within the content section of a Dialog. This section is a minimum of 120px tall and has varying widths for the different sizes of Dialogs.
```

File: src/_areas/design-system/components/dialogs.md [hunk 5]
```diff
@@ -1,122 +1,118 @@
...
 
-H1 title should not span more than 1 line.
+1. **Title:** Ideally, the H1 title in a Dialog should not span more than one line. The title gives the user an understanding of the message or decision they are being presented with. 
+2. **Content area:** More detailed information is contained within the content section of a Dialog. This section is a minimum of 120px tall and has varying widths for the different types of Dialogs.  
+3. **Footer:** The footer is where all the actions are located within a Dialog. Optionally, the footer may remain sticky when the length of the Dialog exceeds the height of the viewport so that the available actions remain visible to the user at all times. More information on scrolling and sticky footers can be found in behavior and best practices.
 
-<br><br>
+* Primary actions are located on the far right
+* Secondary actions are to the left of the primary action
+* Tertiary actions are located on the far left
```
Comment (juliexiong) [inline src/_areas/design-system/components/dialogs.md:32]:
```suggestion
    * Primary actions are located on the far right
    * Secondary actions are to the left of the primary action
    * Tertiary actions are located on the far left  
```

File: src/_areas/design-system/components/dialogs.md [hunk 6]
```diff
@@ -1,122 +1,118 @@
...
-### Scrolling
+- - -
 
-Best practice is to avoid the need for a dialog to scroll. When scrolling is required, the entire dialog screen will scroll. Remember the to keep 36px padding between the viewport and the dialog when it first appears to the user.
+## Alternate considerations
 
-Background contents that are underneath the dark overlay will remain static while the dialog scrolls.
+* Consider a separate page when the content begins to exceed the height of the viewport in order to avoid scrolling. 
+* Consider using a separate page with a focused state when there are form fields.
+* Consider a Popover or Well for shorter messages that do not interrupt the user's workflow.
```
Comment (juliexiong) [inline src/_areas/design-system/components/dialogs.md:118]:
```suggestion
* Consider a [Popover](/design-system/components/popovers/) or [Well](/design-system/components/wells/) for shorter messages that do not interrupt the user's workflow.
```

File: src/_areas/design-system/components/dialogs.md (hunk 3 above)
Comment (dave52) [reply src/_areas/design-system/components/dialogs.md:28]: Just updated the disclaimer text so it's wrapped by `<docs-note>` (which is also now available in Booster)

**Pull request merged**